### ISSUE TITLE
[InputForm.json]: User is unable to submit a form in which a name has an initial in it.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7429]
https://microsoft.visualstudio.com/OS/_workitems/edit/38003960

**Comment by licanhua** [comment]
It's a sample code, need to update inputForm.json.
